app/scripts/templates/settings/two_step_authentication.mustache

```diff
@@ -20,7 +20,8 @@
 
     <div class="settings-unit-details">
         <p>
-            {{#t}}Add a layer of security to your account by requiring security codes from one of these authentication apps.{{/t}}
+            {{#t}}Add a layer of security to your account by requiring security codes from one of {{/t}}
+            <a class="totp-support-link" target="_blank" href="{{totpSupportUrl}}">{{#t}}these authentication apps.{{/t}}</a>
```

Comment:
Splitting a string like this will be confusing to translators

Reply:
Would it make sense to put this in an `unsafeTranslate` tag?

Reply:
You can do something like this:
```html
{{#unsafeTranslate}}Add a layer of security to your account by requiring security codes from one of <a %(totpSupportAttributes)s>these authentication apps.</a>{{/unsafeTranslate}}
```
Then pass in a list of the attributes as a context variable.